### PR TITLE
Homepage codesnippets

### PR DESCRIPTION
**Pull request by itsmebipul00**

### CODE REVIEW

File: Components/ResposiveImages/ResponsiveImages.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .picture-information {
   margin-top: 0 !important;
+  margin-bottom: 0 !important;
```
Comment (sudipkundu999) [inline Components/ResposiveImages/ResponsiveImages.css:3]: use shorthand

Comment (itsmebipul00) [reply Components/ResposiveImages/ResponsiveImages.css:3]: yes sir

Comment (itsmebipul00) [reply Components/ResposiveImages/ResponsiveImages.css:3]: btw cant use only for resetting top and bottom
https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#margin_and_padding_properties

**Review by tonymike99 — APPROVED** [review]
Great work! Good to go.

**Review by sudipkundu999 — APPROVED** [review]
Looks good!
One suggestion: Try to give the same height it'll look consistent.
bipul

File: Components/ResposiveImages/ResponsiveImages.css (hunk 1 above)
Comment (sudipkundu999) [reply Components/ResposiveImages/ResponsiveImages.css:3]: okay! so, "margin: 0 auto !important" won't work?

**Review by rutu14 — APPROVED** [review]
Good to merge.

File: Components/ResposiveImages/ResponsiveImages.css (hunk 1 above)
Comment (itsmebipul00) [reply Components/ResposiveImages/ResponsiveImages.css:3]: i dont want left right auto

Comment (itsmebipul00) [reply Components/ResposiveImages/ResponsiveImages.css:3]: here 0 would be top-bottom and auto would be left right so

Comment (sudipkundu999) [reply Components/ResposiveImages/ResponsiveImages.css:3]: ok cool! Go ahead!

**Review by zoef-yt — APPROVED** [review]
Looks good to me. Good to merge.

**Pull request merged**